sdk/web-pubsub/web-pubsub/test/integration.spec.ts

```diff
@@ -3,7 +3,7 @@
 
 import { WebPubSubServiceClient, odata } from "../src/index.js";
 import { isLiveMode, assertEnvironmentVariable } from "@azure-tools/test-recorder";
-import ws from "ws";
+import { WebSocket } from "ws";
```

Comment:
This should have worked too:
```suggestion
import * as ws from "ws";
```